application/actions/requests.go

```diff
@@ -1,26 +1,28 @@
 package actions
 
 import (
+	"context"
 	"errors"
 
 	"github.com/silinternational/wecarry-api/api"
 	"github.com/silinternational/wecarry-api/models"
 )
 
-func convertRequestToAPIType(request models.Request) (api.Request, error) {
+func convertRequestToAPIType(c context.Context, request models.Request) (api.Request, error) {
```

Comment:
nit: it seems like we often use ctx for context, but sometimes c. Do we care about consistency in naming, or does that not really matter?

Reply:
Yes, I think it matters, especially when we can easily get mixed up between `buffalo.Context` and `context.Context`. I believe the norm is `ctx` for `context.Context` and `c` for `buffalo.Context` (FWIW, I would like to see `bc` for `buffalo.Context` but that should probably not be changed at this point in time.)